src/stores/CheckoutController.ts

```diff
@@ -172,3 +179,43 @@ export const paypalPayment = async (successUrl: string, cancelUrl: string, error
     });
   });
 };
+
+export const getSubscriptionSwitches = async (): Promise<unknown> => {
+  const { cleengSandbox } = useConfigStore.getState().getCleengData();
+  const { user, subscription, auth } = useAccountStore.getState();
+
+  if (!user || !subscription || !auth) return;
+  const response = await checkoutService.getSubscriptionSwitches(
```

Comment:
The `InPlayer` service doesn't contain the `getSubscriptionSwitches` method. I think we don't have to throw an error here, but early return so that the `offerSwitches` state will remain empty.

```suggestion
  if (!('getSubscriptionSwitches' in checkoutService) return;
  
  const response = await checkoutService.getSubscriptionSwitches(
```

In the Payment component, we can hide the "upgrade" button so that it still works with the InPlayer integration.